File: app/models/entities/Candidate.go
package entities

//Candidate структура сущности кандидата
type Candidate struct {
	ID          int64  `json:"ID"`           // идентификатор
	Lastname    string `json:"lastname"`     // фамилия
	Firstname   string `json:"firstname"`    // имя
	Middlename  string `json:"middlename"`   // отчество
	PhoneNumber string `json:"phone_number"` // номер телефона
	Email       string `json:"email"`        // адрес электронной почты
	BirthDate   string `json:"birthdate"`    // дата рождения
	Gender      string `json:"gender"`       // пол
}

File: public/js/src/models/entities/Candidate.js
//Класс Кандидат
export class Candidate{
    constructor(ID,lastname,firstname,middlename,birthdate,phone_number,email,gender){
        this.ID = ID;                     //ID
        this.lastname = lastname;         //Фамилия кандидата
        this.firstname = firstname;       //Имя кандидата
        this.middlename = middlename;     //Отчество кандидата
        this.birthdate = birthdate;       //Дата рождения кандидата
        this.phone_number = phone_number; //Номер телефона кандидата
        this.email = email;               //Адрес электронной почты кандидата
        this.gender = gender;             //Пол кандидата
    }
};

